jetbrains-core/src/software/aws/toolkits/jetbrains/services/apprunner/actions/ServiceUrlActions.kt

```diff
@@ -14,7 +16,19 @@ import java.awt.datatransfer.StringSelection
 
 class CopyServiceUrlAction : SingleResourceNodeAction<AppRunnerServiceNode>(message("apprunner.action.copyServiceUrl")), DumbAware {
     override fun actionPerformed(selected: AppRunnerServiceNode, e: AnActionEvent) {
-        CopyPasteManager.getInstance().setContents(StringSelection(selected.service.serviceUrl()))
+        CopyPasteManager.getInstance().setContents(StringSelection(selected.service.urlWithProtocol()))
         ApprunnerTelemetry.copyServiceUrl(selected.nodeProject)
     }
 }
+
+class OpenServiceUrlAction internal constructor(private val launcher: BrowserLauncher) :
+    SingleResourceNodeAction<AppRunnerServiceNode>(message("apprunner.action.openServiceUrl")), DumbAware {
```

Comment:
I have been trying to stop putting the messages into the constructor since they do not localize.

You can leave it blank and add `apprunner.service.openServiceUri.text=Open Service URL` in the messages bundle

Reply:
done